spec/unit/puppet/provider/powershell_spec.rb

```diff
@@ -0,0 +1,126 @@
+#! /usr/bin/env ruby
+require 'spec_helper'
+require 'puppet/type'
+require 'puppet/type/base_dsc'
+
+describe Puppet::Type.type(:base_dsc).provider(:powershell) do
+
+  # Override initialize so we don't call the symlink set
+  class Puppet::Type::Base_dsc::ProviderPowershell
+    def initialize(value={})
+      super(value)
+    end
+  end
```

Comment:
I'm hoping this doesn't have an effect on the regular use of Puppet since this is in the specs.